test/models/GetPenaltyDetailsSpec.scala

```diff
@@ -175,6 +177,8 @@ class GetPenaltyDetailsSpec extends AnyWordSpec with Matchers {
            "penaltyStatus": "A",
            "penaltyAmountPaid": 1001.45,
            "penaltyAmountOutstanding": 99.99,
+           "penaltyAmountPosted": 1001.45,
```

Comment:
This is an accruing penalty, therefore there would be no posted amount. No outstanding and paid amount also